src/patternfly/components/JumpLinks/jump-links.scss

```diff
@@ -10,7 +10,7 @@
   --pf-c-jump-links--m-vertical__list--PaddingLeft: 0;
   --pf-c-jump-links__list--FlexDirection: row;
   --pf-c-jump-links--m-vertical__list--FlexDirection: column;
-  --pf-c-jump-links__list--before--BorderColor: var(--pf-global--BorderColor--100);
+  --pf-c-jump-links__list--before--BorderColor: transparent;
```

Comment:
This removes the border all together 😄 We only want to remove from nested lists.